src/lib/testFlags.ts

```diff
@@ -59,7 +59,7 @@ class TestFlags {
   }
 
   get enableStaticTyping() {
-    return !!this.queryParams.statictyping;
+    return !!this.queryParams.statictyping || isDev;
```

Comment:
can we modify this so that setting statictyping=false actually works in dev though? It's a bit unfortunate we can't quickly check to see if a bug is caused by static typing on staging/dev in the current setup.

Reply:
added a string literal check for `false`

Reply:
AHH you already got to it

Reply:
@aforaleka did, I don't even know if she saw my comment here or how this coordination happened.